app/src/main/java/org/sagebionetworks/research/mpower/tracking/model/TrackingSection.java

```diff
@@ -0,0 +1,41 @@
+package org.sagebionetworks.research.mpower.tracking.model;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
+
+/**
+ * A TrackingSection represents a grouping of TrackingItems used by the UI to display the items in a logical way.
+ */
+@AutoValue
```

Comment:
I did some research on AutoValue.  Am I correct in that it's a way to get auto-generated equals(), toString(), and hashCode() functionality?